src/test/java/com/wikia/webdriver/testcases/mcfootertests/EnAnonMixedContentFooterTests.java

```diff
@@ -0,0 +1,91 @@
+package com.wikia.webdriver.testcases.mcfootertests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.mcfooter.MixedContentFooter;
+
+import org.testng.annotations.Test;
+
+@Test (groups = {"EnAnonMixedContentFooter"})
+@Execute(onWikia = "gameofthrones", asUser = User.ANONYMOUS)
+public class EnAnonMixedContentFooterTests extends NewTestTemplate{
+
+  @Test
+   public void mcFooterIsPresentOnENwiki() {
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isMcfooterPresent());
+  }
+
+  @Test
+   public void exploreWikisCardIsPresentOnENwiki(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isExploreWikisCardPresent());
+  }
+
+  @Test
+  public void discussionsCardIsPresentOnENwikiWithDiscussions(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isDiscussionsCardPresent());
+  }
+
+  @Test
+  @Execute(onWikia = "agas")
+  public void discussionsCardIsPresentOnENwikiWithEmptyDiscussions(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isDiscussionsCardPresent());
+  }
+
+  @Test
+  @Execute(onWikia = "glee")
```

Comment:
is it safe to assume that glee will have always disabled disucssions?

Reply:
It is not but this is only temporary solution. I will crate wikis with given MCFooter setup so MCFooter tests will run only on that dedicated wikis.